circle.yml

```diff
@@ -28,9 +30,11 @@ test:
     - ./configure -i
   override:
     - env CXXFLAGS="-Werror" make test
+    - valgrind --leak-check=full --show-leak-kinds=definite,indirect,possible --error-exitcode=1 ./bin/test-libdrafter 2> memcheck.log
```

Comment:
I don't think this would be useful in CircleCI, with word-wrapping it's very difficult to read.

Example:

![screen shot 2016-04-05 at 13 54 07](https://cloud.githubusercontent.com/assets/44164/14280877/e50ff8ae-fb35-11e5-92ad-749732a0821a.png)